{"uid":"e419bd01bd109f98","name":"test_new_pkg_format_preferred","fullName":"tests.test_index#test_new_pkg_format_preferred","historyId":"a6b8a31be9fd54a49616cf177c73794b","time":{"start":1673022299596,"stop":1673022299736,"duration":140},"description":"Test that in one pass, the .conda file is extracted before the .tar.bz2, and the .tar.bz2 uses the cache","descriptionHtml":"<p>Test that in one pass, the .conda file is extracted before the .tar.bz2, and the .tar.bz2 uses the cache</p>\n","status":"failed","statusMessage":"AssertionError: Expected 'extract' to be called once. Called 0 times.","statusTrace":"Traceback (most recent call last):\n  File \"D:\\a\\conda-build\\conda-build\\tests\\test_index.py\", line 769, in test_new_pkg_format_preferred\n    cph_extract.assert_called_once_with(test_package_path + \".conda\", mock.ANY, \"info\")\n  File \"C:\\Users\\runneradmin\\Miniconda3\\lib\\unittest\\mock.py\", line 212, in assert_called_once_with\n    return mock.assert_called_once_with(*args, **kwargs)\n  File \"C:\\Users\\runneradmin\\Miniconda3\\lib\\unittest\\mock.py\", line 930, in assert_called_once_with\n    raise AssertionError(msg)\nAssertionError: Expected 'extract' to be called once. Called 0 times.","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"tmp_path_factory","time":{"start":1673019499430,"stop":1673019499430,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"pytestconfig","time":{"start":1673020332445,"stop":1673020332445,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"monkeypatch","time":{"start":1673022299596,"stop":1673022299596,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"default_testing_config","time":{"start":1673022299596,"stop":1673022299596,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"mocker","time":{"start":1673022299596,"stop":1673022299596,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"testing_workdir","time":{"start":1673022299596,"stop":1673022299596,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"testing_config","time":{"start":1673022299596,"stop":1673022299596,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"tmp_path","time":{"start":1673022299596,"stop":1673022299596,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"tmpdir","time":{"start":1673022299596,"stop":1673022299596,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"description":"Test that in one pass, the .conda file is extracted before the .tar.bz2, and the .tar.bz2 uses the cache","status":"failed","statusMessage":"AssertionError: Expected 'extract' to be called once. Called 0 times.","statusTrace":"Traceback (most recent call last):\n  File \"D:\\a\\conda-build\\conda-build\\tests\\test_index.py\", line 769, in test_new_pkg_format_preferred\n    cph_extract.assert_called_once_with(test_package_path + \".conda\", mock.ANY, \"info\")\n  File \"C:\\Users\\runneradmin\\Miniconda3\\lib\\unittest\\mock.py\", line 212, in assert_called_once_with\n    return mock.assert_called_once_with(*args, **kwargs)\n  File \"C:\\Users\\runneradmin\\Miniconda3\\lib\\unittest\\mock.py\", line 930, in assert_called_once_with\n    raise AssertionError(msg)\nAssertionError: Expected 'extract' to be called once. Called 0 times.","steps":[],"attachments":[{"uid":"6b89695782ece4b8","name":"log","source":"6b89695782ece4b8.txt","type":"text/plain","size":11680}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"monkeypatch::0","time":{"start":1673022299736,"stop":1673022299736,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"mocker::0","time":{"start":1673022299736,"stop":1673022299736,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"testing_workdir::return_to_saved_path","time":{"start":1673022299736,"stop":1673022299736,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_index"},{"name":"host","value":"fv-az195-793"},{"name":"thread","value":"3544-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_index"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"e419bd01bd109f98.json","parameterValues":[]}